docusaurus/docs/iOS/uikit/views/thread-header-view.md

```diff
@@ -5,7 +5,7 @@ title: ChatThreadHeaderView
 import ComponentsNote from '../../common-content/components-note.md'
 import Properties from '../../common-content/reference-docs/stream-chat-ui/chat-thread/chat-thread-header-view-properties.md'
 
-This component is responsible to display the information in the header of a thread. By default, it is rendered in the `navigationItem.titleView` of the [`ChatThreadVC`](../thread).
+This component is responsible to display the information in the header of a thread. By default, it is rendered in the `navigationItem.titleView` of the [`ChatThreadVC`](../../components/thread/).
```

Comment:
e.g. this one

Reply:
it opens `https://github.com/GetStream/stream-chat-swift/blob/23f48f32193f9bfb508be33aab15dd88d9b63bf5/docusaurus/docs/iOS/components/thread`

but I think it should open `https://github.com/GetStream/stream-chat-swift/blob/23f48f32193f9bfb508be33aab15dd88d9b63bf5/docusaurus/docs/iOS/uikit/components/thread.md` 🤔

Reply:
Hm, okay, will take a look when deployed 👍